docs/site/Using-strong-error-handler.md

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Using strong-error-handler'
+lang: en
+keywords: LoopBack 4, error handling
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Using-strong-error-handler.html
+---
+
+# strong-error-handler
+
+As the part of the dependencies of
+[`@loopback/rest`](https://github.com/strongloop/loopback-next/tree/master/packages/rest),
+package `strong-error-handler` is an error handler for use in both development
+(debug) and production environments.
+
+In LoopBack 4, the request handling process starts with the app's
+[sequence](https://loopback.io/doc/en/lb4/Sequence.html), a simple class with
+five injected helper methods in the constructor. It is the gatekeeper of all
+requests to the app. Errors are handled by one of the Sequence actions,
+[`reject`](Sequence.md#handling-errors), which calls `strong-error-handler`
+package to send back an HTTP response describing the error.
+
+LB4 apps require 3.x versions of `strong-error-handler`.
+
+## Usage
+
+As mentioned above, errors are handled by the `reject` action of the app's
+[sequence](https://loopback.io/doc/en/lb4/Sequence.html). By default, `reject`
```

Comment:
Nitpick: a suggestion for rephrasing the usage example :)

---
You can configure the error handler's behaviors by setting the error writer options.
For example, by default, `reject` omits details from error responses to prevent leaking sensitive information. But `reject` also can return full error stack traces and internal details of any error objects to the client in the HTTP responses. This is controlled by the option property `debug`:

```ts
app.bind(RestBindings.ERROR_WRITER_OPTIONS).to({debug: true});
```

Once the `debug` flag is set, `strong-error-handler` would handle such options.